packages/graphql-hooks/src/GraphQLClient.js

```diff
@@ -99,21 +101,53 @@ class GraphQLClient {
     }
   }
 
-  request(operation, options = {}) {
-    return this.fetch(this.url, {
+  getFetchOptions(operation, fetchOptionsOverrides = {}) {
```

Comment:
Let's give kudos to @jaydenseric and link to the original source of this https://github.com/jaydenseric/graphql-react/blob/master/src/universal/graphqlFetchOptions.mjs 🙌